maquetado/detail.html

```diff
@@ -1,14 +1,17 @@
 <!doctype html>
 
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>Books</title>
-    <meta name="description" content="training">
-    <meta name="author" content="Wolox">
-    <link rel="stylesheet"  type="text/css" href="application.css">
-  </head>
+<head>
+  <meta charset="utf-8">
+  <title>Books</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
```

Comment:
The changes in the head section are note necessary

Reply:
Done